config/test.exs

```diff
@@ -27,7 +27,7 @@ config :logger, level: :warning
 # Initialize plugs at runtime for faster test compilation
 config :phoenix, :plug_init_mode, :runtime
 
-config :wanda, Wanda.Catalog, catalog_path: "test/fixtures/catalog"
+config :wanda, Wanda.Catalog, catalog_paths: ["/opt/trento-checks", "test/fixtures/catalog"]
```

Comment:
I think you're right, I just nurture a great love for /opt because of old times and old software I used, but it's something really really personal :smile: 

Also I remember a lot of people in the team mentioning the fact that we also have a `/var` "namespace". I actually like `/usr/share` tho :sweat_smile:

Reply:
I like `/usr/share/trento/checks{,.d}` :+1:

I would also keep `/opt/trento/checks` when we decide we're ready to accept users loading their checks into the system :+1: